test/e2e/run_device_test.go

```diff
@@ -152,9 +155,12 @@ var _ = Describe("Podman run device", func() {
 	})
 
 	It("podman run cannot access non default devices", func() {
-		session := podmanTest.Podman([]string{"run", "-v /dev:/dev-host", ALPINE, "head", "-1", "/dev-host/kmsg"})
+		// Unlikely to happen but do not read any data from the device (thus the -n 0), because there is a rare
+		// race condition that happens so the test would fail for the rare race condition instead of a failure
+		// reported by open(). More details: https://github.com/containers/podman/issues/23882.
+		session := podmanTest.Podman([]string{"run", "-v /dev:/dev-host", ALPINE, "head", "-n0", "/dev-host/kmsg"})
```

Comment:
> creating named volume " /dev": running volume create option: names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*: invalid argument
```suggestion
		session := podmanTest.Podman([]string{"run", "-v", "/dev:/dev-host", ALPINE, "head", "-n0", "/dev-host/kmsg"})
```

Oh well that is exactly why we must have error message checks. This was never testing what it said it was testing...